x/ibc/core/02-client/client/cli/query.go

```diff
@@ -222,12 +221,11 @@ func GetCmdNodeConsensusState() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			state, height, err := utils.QueryNodeConsensusState(clientCtx)
+			state, _, err := utils.QueryNodeConsensusState(clientCtx)
 			if err != nil {
 				return err
 			}
 
-			clientCtx = clientCtx.WithHeight(height)
```

Comment:
Why is this no longer needed?

Reply:
`PrintProto` doesn't print the context height. This code was likely introduced after confusion of looking at the previously existing rest code. The helper function for returns rest responses would inject the height into the response based on the context height. Since queries of a context height of 0 use the latest height, this was a necessary step to returning the correct height

Again, `PrintProto` doesn't use the context height